### PR TITLE
Blood: Rename various functions

### PR DESCRIPTION
**Pull request by carnivoroussociety**
This rename sub_5BCF4() to ResetCheats(), and sub_746D4() to fxSpawnBloodPod(). It also renames CDemo's at59ef to nDemosFound.

### CODE REVIEW

File: source/blood/src/fx.cpp [hunk 1]
```diff
@@ -289,7 +289,7 @@ void fxSpawnBlood(spritetype *pSprite, int a2)
     }
 }
 
-void sub_746D4(spritetype *pSprite, int a2)
+void fxSpawnBloodPod(spritetype *pSprite, int a2)
```
Comment (zZeck) [inline source/blood/src/fx.cpp:292]: what's a blood pod?

Comment (carnivoroussociety) [reply source/blood/src/fx.cpp:292]: Well fxSpawnBloodPod() operates the same as fxSpawnBlood() but for green/red pod enemies.

Comment (zZeck) [reply source/blood/src/fx.cpp:292]: OOOOOH. Maybe fxSpawnPodBlood() would be better? Could just be me.

Comment (carnivoroussociety) [reply source/blood/src/fx.cpp:292]: I agree, fixed.

**Comment by sirlemonhead** [comment]
Thanks!

**Pull request merged**